### ISSUE TITLE
Doesn't work in hipchat

### ISSUE THREAD

**Issue by dylanh724** [hubot-scripts/hubot-seen#8]
Seems broken in hipchat

**Comment by dylanh724** [comment]
@ dbg "seen.add #{clean user} on #{channel}"
returning undefined on channel.

Then when asked if seen, never seen.

**Comment by patcon** [comment]
Yep sorry, only ever used it for IRC (on slack as well, but using IRC adapter). I'm totally down for merging a PR if you can get it going though, just not able to do it myself :)

**Comment by dylanh724** [comment]
Sorry, I'm not quite sure how to do it, I was just giving direction
